src/olympia/files/management/commands/update_permissions_from_mc.py

```diff
@@ -24,10 +22,9 @@ def handle(self, *args, **options):
 
         central_url = settings.WEBEXT_PERM_DESCRIPTIONS_URL
         locales_url = settings.WEBEXT_PERM_DESCRIPTIONS_LOCALISED_URL
+        amo_locales = [l for l in settings.AMO_LANGUAGES if l != 'en-US']
```

Comment:
It's the debug locale :) https://addons-dev.allizom.org/dbg/firefox/

It should not be enabled in prod, but it'll make noise on dev.